### DISCUSSION TITLE
Sharing Figma Example of the New Listbox Shadow + Border

### DISCUSSION THREAD

**Discussion by ckizer** [tailwindlabs/tailwindcss#13035] [General]
Love the new design. I need to be able to use some of the dropdown listboxes in Figma mockups to the team. Can you share an example Figma file of how you created the dropdown effect in Figma? I've been getting close by using a gradient border, but that feels like cheating. **I would love to get as close as possible to the great work you did.** Paid TailwindsUI user who loves what you've done with Catalyst!

Your design is on top, mine is on body.
![CleanShot 2024-02-26 at 09 36 29@2x](https://github.com/tailwindlabs/tailwindcss/assets/191738/6b4a85f4-0d00-4986-99d4-0e2aad65ae0e)

![Uploading CleanShot 2024-02-26 at 09.35.00@2x.png…]()

**Comment by autonomous-agent-7** [comment]
@autonomous-agent-7 team-lead → routing to **project-manager** for triage.

Please review this feature request and determine:
1. Does it fit within the current epic scope?
2. Dependencies on existing work (#4, #5)?
3. Create a child Discussion if accepted, or close if declined.

/next → Reply when triage complete.
